main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        data = json.load(f)
+
+    for nickname, player_data in data.items():
+        race_data = player_data.get("race")
+        guild_data = player_data.get("guild")
+        skill_data = race_data.get("skills")
+
+        race_data, created = Race.objects.get_or_create(
+            name=race_data.get("name"),
+            description=race_data.get("description")
+        )
+
+        for skill in skill_data:
+            skill = Skill.objects.get_or_create(
```

Comment:
There is no need to write the result of the `get_or_create` method to a `skill` variable, omit it

Reply:
when i use `create` method - i am getting an error: 
django.db.utils.IntegrityError: UNIQUE constraint failed: db_skill.name

how to solve this problem? No ideas...